tests/unit/AddStaffNotationDialog.spec.ts

```diff
@@ -0,0 +1,246 @@
+import Vue from 'vue'
+import Vuetify from 'vuetify'
+import flushPromises from 'flush-promises'
+import sinon from 'sinon'
+import { shallowMount, mount } from '@vue/test-utils'
+import { getVuexStore } from '@/store'
+import axios from '@/axios-auth'
+import { AddStaffNotationDialog } from '@/components/dialogs'
+import { CourtOrderPoa } from '@bcrs-shared-components/court-order-poa'
+
+Vue.use(Vuetify)
+
+const vuetify = new Vuetify({})
+const store = getVuexStore()
+describe('AddStaffNotationDialog', () => {
+  it('renders the page contents correctly', () => {
+    const wrapper = shallowMount(AddStaffNotationDialog,
+      {
+        propsData: {
+          dialog: true,
+          itemName: 'Test',
+          attach: '#parent-page'
+        },
+        store,
+        vuetify
```

Comment:
You may not need "vuetify" here since you're shallow-mounting the component. I'm not sure. Just an FYI - no need to change.